src/main/webapp/history.js

```diff
@@ -40,6 +42,11 @@ async function loadContent() {
   const photosResponse = await fetch('/photos');
   const photos = await photosResponse.json();
 
+  if (photos.length == 0) {
+    contentDiv.innerHTML =
+        'You didn\'t upload any photo using this account yet.';
+  }
```

Comment:
Would it make sense to return here, so that it is clear that we won't add anything to the content after that?

Reply:
Right, thank you! I added a return there.